FrontEnd/src/components/ProfilePage/FormComponents/GeneralInfo.js

```diff
@@ -191,8 +192,11 @@ const GeneralInfo = (props) => {
                 if (response.status === 200) {
                     const updatedProfileData = await response.json();
                     profileMutate(updatedProfileData);
-                } else {
-                    console.error('Помилка');
+                } else if (response.status === 400 ) {
+                    const errorData = await response.json();
+                    if (errorData.edrpou[0] === 'profile with this edrpou already exists.') {
+                        toast.error('Компанія з таким ЄДРПОУ вже існує');
+                    }
```

Comment:
I think we want to add else clause with an `unknown error'

Reply:
Done